.github/workflows/ci-docker-compose-integration.yml

```diff
@@ -118,6 +118,10 @@ jobs:
         run: |
           set +e
           docker ps -a
+          echo ::group::MIGRATION_LOGS
+          docker logs --tail=10000 compose-standalone-migrations-1
+          docker logs --tail=10000 compose-insights-migrations-1
+          echo ::endgroup::
```

Comment:
It may be valid to add this to other files within the `.github/workflows` folder as well. This is just 1 particular check where I hit it, so this was sufficient to surface errors for my debugging.